host_applications/linux/apps/raspicam/imx219_modes.h

```diff
@@ -53,10 +53,10 @@ struct sensor_regs imx219_8MPix[] =
       {0x0157, 0x00},         // Analog Gain
       {0x0158, 0x00},         // Digital Gain [15:8]
       {0x0159, 0x00},          // Digital Gain [7:0]
-      {0x015A, 0x01},      // Shutter/Integration Time [15:8]
-      {0x015B, 0x00},      // Shutter/Integration Time [7:0]
-      {0x0160, 0x09},      // Frame Length [15:8]
-      {0x0161, 0xC8},      // Frame Length [7:0]
+      {0x015A, 0x09},      // Shutter/Integration Time [15:8]
+      {0x015B, 0xBD},      // Shutter/Integration Time [7:0]
+      {0x0160, 0x0D},      // Frame Length [15:8]
+      {0x0161, 0x09},      // Frame Length [7:0]
```

Comment:
These are likely to get overwritten from the command line, but those look like more sensible defaults.